config/jobs/periodic/docker-in-docker/periodic-build-docker.yaml

```diff
@@ -0,0 +1,52 @@
+periodics:
+  - name: periodic-build-docker
+    cluster: k8s-ppc64le-cluster
+    decorate: true
+    reporter_config:
+      slack:
+        channel: 'prow-job-notifications'
+        job_states_to_report:
+         - success
+        report_template: 'Job {{.Spec.Job}} of type {{.Spec.Type}} ended with state {{.Status.State}}. <!subteam^S01E9J0G3EU> <{{.Status.URL}}|View logs>'
+    decoration_config:
+      gcs_configuration:
+        bucket: s3://ppc64le-docker
+        path_strategy: explicit
+      gcs_credentials_secret: s3-auth
+      timeout: 20h
+    extra_refs:
+    - org: ppc64le-cloud
+      repo: docker-ce-build
+      base_ref: main
+      workdir: false
```

Comment:
I wanted /workspace to be the work directory, as specified in the Dockerfile but even though I put "workdir: false" the work directory was always overwritten and it would end up being docker-ce-build.

Reply:
@florencepascual As you mention, that work directory is always being `docker-ce-build` inspite of true/false in `workdir`, Have you written your scripts taking this into consideration?

Reply:
@florencepascual Can you also point me to the part of code that needs the work directory to be `/workspace`?

Reply:
First, everything is clearer when all "git clone" I make (docker-ce-packaging and containerd-packaging) and files are in the /workspace instead of the docker-ce-build directory.
Second, I sometimes use docker containers (to build the static packages and also to test the dynamic and static packages). And for these packages, the workdir is /workspace.
https://github.com/ppc64le-cloud/docker-ce-build/blob/main/build.sh#L130
https://github.com/ppc64le-cloud/docker-ce-build/blob/main/test.sh#L116
https://github.com/ppc64le-cloud/docker-ce-build/blob/main/test.sh#L219